### DISCUSSION TITLE
System Tile/Bottom Online Image shouldn't be visible when menu isn't open

### DISCUSSION THREAD

**Discussion by SupDos** [ppy/osu#26199] [Ideas] [answered]
In stable, the menu will fade out after a bit, hiding the bottom image with it
![image](https://github.com/ppy/osu/assets/6813986/5df3b574-f473-4d49-9ede-8a30d0bc0b96)

In lazer however, when no menu is opened, the image is still visible
![image](https://github.com/ppy/osu/assets/6813986/a7795af2-7924-4fbe-8787-4f9f6fb05aab)

I think it should hide itself when no menu is open, and appear after clicking the osu cookie to at least sort of match stable's appearance

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/pull/26221
